### DISCUSSION TITLE
How to use field selectors for PODs?

### DISCUSSION THREAD

**Discussion by zdrux** [openshift/openshift-client-python#136] [Q&A] [answered]
I am new to Python and this module, so please bare with me...

I am trying to get a list of pods in Running state and can't figure out proper way to use field selector

running_selector = "status.phase=Running"
pod_obj = oc.selector("pods", field_selectors=running_selector)

...the above doesn't seem to work, is anyone able to help?

**Comment by bradmwilliams** [comment, marked as answer]
Hi @zdrux,
You are not too far off actually!  This issue is that `running_selector` should be a dict:
```python
running_selector = {"status.phase": "Running"}
```
